azure-iot-device/tests/common/pipeline/test_pipeline_flow.py

```diff
@@ -18,6 +18,7 @@
     assert_callback_failed,
     assert_callback_succeeded,
 )
+from .fixtures import FakeOperation
```

Comment:
>FakeOperation [](start = 22, length = 13)

Lately I've been trying to keep class definitions used for fixtures to be internally defined within the fixture itself, and not pass around the class definition.

However, here, given that there are multiple op fixtures, including ones defined exclusively for this file (e.g. arbitrary_worker_op), I see why this is necessary